ArchiSteamFarm/Storage/GlobalConfig.cs

```diff
@@ -319,8 +321,15 @@ public sealed class GlobalConfig {
 				return (false, string.Format(CultureInfo.CurrentCulture, Strings.ErrorConfigPropertyInvalid, nameof(OptimizationMode), OptimizationMode));
 			}
 
-			if (!string.IsNullOrEmpty(SteamMessagePrefix) && (SteamMessagePrefix!.Length > Bot.MaxMessagePrefixLength)) {
-				return (false, string.Format(CultureInfo.CurrentCulture, Strings.ErrorConfigPropertyInvalid, nameof(SteamMessagePrefix), SteamMessagePrefix));
+			if (!string.IsNullOrEmpty(SteamMessagePrefix)) {
+				int sizeInBytes = Encoding.UTF8.GetByteCount(SteamMessagePrefix);
+
+				// Take into account newlines weight, if somebody is really awkward enough to put some in
+				sizeInBytes += SteamMessagePrefix.Count(c => c == '\n') * (SteamChatMessage.NewlineWeight - 1);
```

Comment:
No, if user has `\n` in the json but we count `\r\n` because he's using that file on Windows then we'll calculate it wrong.

You could now argue that `\r\n` should also count as one newline and not like currently to one newline + 1 character (`\r`) but it's really not my concern, nobody should use newlines in message prefix anyway, and if somebody does then I don't care being 1 byte lower on each newline in the calculation, as long as it's below the maximum limit.

Reply:
(I took my own conclusion into account anyway and now it's 1:1 correct calculation, regardless of circumstances, so nobody can even complain we're 1 byte short, lol)